Packs/Jira/Integrations/JiraV3/README.md

```diff
@@ -2941,3 +2941,57 @@ In the *jira-edit-issue* command:
 In the *jira-get-issue* command:
 
 * *get_attachments* - Supplying this argument with the value `true` will return the attachments found in the specified issue as `Entry Info File`, and not as `File`.
+
+### jira-get-user-info
```

Comment:
Lets also have a command usage example, and a context example (see other command's descriptions above).